test/e2e/security/request_smuggling.go

```diff
@@ -51,8 +51,8 @@ server {
 		f.UpdateNginxConfigMapData("http-snippet", snippet)
 
 		ing := framework.NewSingleIngress(host, "/", host, f.Namespace, framework.EchoService, 80, map[string]string{
-			"nginx.ingress.kubernetes.io/auth-signin": "https://httpbin.org/uuid",
-			"nginx.ingress.kubernetes.io/auth-url":    "https://httpbin.org/basic-auth/user/passwd",
+			"nginx.ingress.kubernetes.io/auth-signin": "https://httpbun.com/bearer/d4bcba7a-0def-4a31-91a7-47e420adf44b",
```

Comment:
why we just don't use internal httpbun here?

Reply:
fair point, didnt think of that!

Reply:
I mean, if we create a service for httpbun, and point to this service ip via NIP, we can still reach it right?

Reply:
Is this pushed? Can't see it :D